tr/tr.go

```diff
@@ -0,0 +1,59 @@
+package tr
+
+import (
+	"encoding/base64"
+	"os"
+	"strings"
+
+	"github.com/leonelquinteros/gotext"
+)
+
+//go:generate go run ../tr/trgen/trgen.go
+
+var Tr = gotext.NewLocale("/usr/share/locale", "en")
```

Comment:
It will not, but since we don't expect external locale files at all for Git LFS, it shouldn't be a problem.  It may be that a Linux distro chooses to ship these externally, but that wouldn't work on Windows anyway since Windows doesn't have a canonical place for them, so again, the fact that this does nothing shouldn't be a problem.